braindecode/models/tcn.py

```diff
@@ -50,12 +48,12 @@ def __init__(
             n_filters=None,
             kernel_size=None,
             drop_prob=None,
-            add_log_softmax=None,
             chs_info=None,
             n_times=None,
             input_window_seconds=None,
             sfreq=None,
             n_in_chans=None,
+            add_log_softmax=True,
```

Comment:
```suggestion
            add_log_softmax=False,
```
As it was default to false before, we should leave the same default here